instrumentation/net_http/lib/opentelemetry/instrumentation/net/http/patches/instrumentation.rb

```diff
@@ -31,7 +31,7 @@ def request(req, body = nil, &block)
                 attributes: attributes,
                 kind: :client
               ) do |span|
-                OpenTelemetry.propagation.inject(req)
+                inject_context_and_extract_data(span, req, body)
```

Comment:
What do you think about keeping the `propagation.inject` call up here, then, on the next line, invoking `safe_execute_hook if instrumentation_config[:request_hook]`. I'm not sure why we'd group together `inject_context_and_extract_data` into a single method (but maybe I'm missing something!).

Reply:
+1

Reply:
Hi @plantfansam @arielvalentin  , the reason I did it was to avoid `Metrics/AbcSize` lint error. For some reason only in Ruby 2.7 it would fail. I'd appreciate any other suggestion you may to workaround this error. Thanks!

Reply:
@plantfansam we should probably take a look at the ABCSize rule and see if we should relax it globally for instrumentations that use prepended modules.

Reply:
Related https://github.com/open-telemetry/opentelemetry-ruby-contrib/issues/66

Reply:
@nozik you can add `# rubocop:disable Metrics/AbcSize` as a comment at the end of the line that Rubocop complains about, and that will fix it!

Reply:
@plantfansam Done!